front/app/containers/Admin/projects/edit/participationContext/components/SurveyInputs.tsx

```diff
@@ -38,52 +38,102 @@ export default injectIntl<Props & InjectedIntlProps>(
       <SectionField>
         <SubSectionTitle>
           <FormattedMessage {...messages.surveyService} />
-          <IconTooltip
-            content={
-              <FormattedMessage
-                {...messages.surveyServiceTooltip}
-                values={{
-                  surveyServiceTooltipLink: (
-                    <StyledA
-                      href={formatMessage(messages.surveyServiceTooltipLink)}
-                      target="_blank"
-                    >
-                      <FormattedMessage
-                        {...messages.surveyServiceTooltipLinkText}
-                      />
-                    </StyledA>
-                  ),
-                }}
-              />
-            }
-          />
         </SubSectionTitle>
         <StyledWarning>
           <FormattedMessage
-            {...messages.hiddenFieldsTip}
+            {...messages.surveyServiceTooltip}
             values={{
-              hiddenFieldsLink: (
-                <a
-                  href={formatMessage(messages.hiddenFieldsSupportArticleUrl)}
+              surveyServiceTooltipLink: (
+                <StyledA
+                  href={formatMessage(messages.surveyServiceTooltipLink)}
                   target="_blank"
-                  rel="noreferrer"
+                  rel="noopener noreferrer"
                 >
-                  {formatMessage(messages.hiddenFieldsLinkText)}
-                </a>
+                  <FormattedMessage
+                    {...messages.surveyServiceTooltipLinkText}
+                  />
+                </StyledA>
               ),
             }}
           />
         </StyledWarning>
-        {Object.keys(surveyProviders).map((provider) => {
+        {Object.keys(surveyProviders).map((provider: TSurveyService) => {
           if (surveyProviders[provider]) {
             return (
-              <Radio
+              <SurveyServiceRadio
                 onChange={handleSurveyProviderChange}
                 currentValue={survey_service}
                 value={provider}
                 name="survey-provider"
                 id={`survey-provider-${provider}`}
-                label={<FormattedMessage {...messages[provider]} />}
```

Comment:
1. Our script to catch unused strings won't catch any strings that are uses here. It searches for messages.key, not for messages[key]
2. This adds an extra type check. By changing it to this format:

```
{
  survey_monkey: formatMessage(messages.survey_monkey),
  survey_xact: formatMessage(messages.survey_xact),
  enalyzer: formatMessage(messages.enalyzer),
  qualtrics: formatMessage(messages.qualtrics),
  smart_survey: formatMessage(messages.smart_survey),
  microsoft_forms: formatMessage(messages.microsoft_forms),
}[provider]
```
the page crashed because survey_xact wasn't in this list at first because it wasn't in the definition of the service types (TSurveyService in projects.ts). And so it caught a bug :)